### ISSUE TITLE
AR localize support

### ISSUE THREAD

**Issue by strangeliu** [strangeliu/ttte-i18n#3]
@iMaz1n many thanks for your work on the ar language support, I found that some strings is missing in the localized files,

account.verify.alert.action
account.verify.alert.message
account.verify.alert.title
common.copyLink
common.tweet
compose.drafts.title
contextAction.url.openWithReadingMode
contextAction.url.openWithSystemBrowser
list.loadMore
lists.create.title
lists.viewDetail
setting.accentColor
setting.appIcon
setting.barButtonFollowAccentColor
setting.linksFollowAccentColor
setting.presenting.avatarLayout
setting.presenting.avatarLayout.alongside
setting.presenting.avatarLayout.inline
setting.presenting.layoutStyle
setting.presenting.layoutStyle.card
setting.presenting.layoutStyle.list
setting.presenting.layoutStyle.timedList
setting.presenting.mediaLayoutStyle
setting.presenting.mediaLayoutStyle.compact
setting.presenting.mediaLayoutStyle.default
setting.presenting.mediaLayoutStyle.fullSize
setting.presenting.mediaLayoutStyle.keepSize
setting.presenting.separator
setting.presenting.separator.useWideSeparator
tweet.action.like
tweet.action.quote
tweet.action.unlike
tweet.content.viewThread
user.profile.action.block
user.profile.action.block.alert.confirm
user.profile.action.mute
user.profile.action.mute.alert.confirm
user.profile.action.unblock
user.profile.action.unblock.alert.confirm
user.profile.action.unmute
user.profile.action.unmute.alert.confirm
user.profile.actions.copyScreenName
user.profile.blocked
user.profile.blockedBy
user.profile.chat
user.profile.followedBy
user.profile.listed
user.unfollow.alert.message

Hope you can make a new PR to complete it,

and I released a new TestFlight version added ar language and RTL support, https://testflight.apple.com/join/GUmL2FT2,

Thanks again for your great work.

**Comment by itsMaz1n** [comment]
you're so welcome,
please can you check testflight copy? got "This beta isn't accepting any new testers right now."
I'll complete missing strings the PULL
thanks

**Comment by strangeliu** [comment]
@iMaz1n Thanks for your PR, https://testflight.apple.com/join/GUmL2FT2 the TestFlight version is open now.
